include/deal.II/lac/vector.templates.h

```diff
@@ -134,12 +135,62 @@ namespace internal
                            tbb::auto_partitioner());
       }
     else if (vec_size > 0)
-      functor(0,vec_size);
 #endif
+      functor(0,vec_size);
```

Comment:
I can change this in a follow-up patch. It's probably not the best place to save four lines of code...